src/domains/chains/utils/definitions/definitions.ts

```diff
@@ -19,12 +20,13 @@ export type Definition = {
     relayerUrl: string,
   },
   urlPathSegment: string,
+  disabled?: boolean,
```

Comment:
I think it'd be better to construct `ChainConfig` so that it's possible to not define either testnet or mainnet, instead of "pretending" to define them by providing deficient data and not even use them anywhere.